magic_numbers.py

```diff
@@ -81,13 +81,20 @@ def _parse_float_name(name: str) -> Union[float, None]:
 	return float(f"{int_part}.{decimal_str}")
 
 def __getattr__(name: str) -> Union[int, float]:
+	is_negative = False
+	if name.startswith('NEGATIVE_') or name.startswith('MINUS_'):
+		is_negative = True
+		if name[0] == 'N':
+			name = name[9:] # remove "NEGATIVE_"
+		elif name[0] == 'M':
+			name = name[6:] # remove "MINUS_"
```

Comment:
one more nit - 9 and 6 here are, ironically, magic numbers. we could avoid this (and simplify in general) by doing `name = name.partition("_")[2]`